shared/hashutil/hash.go

```diff
@@ -0,0 +1,20 @@
+package hashutil
+
+import (
+	"golang.org/x/crypto/sha3"
+)
+
+// Hash defines a function that returns the
+// Keccak-256/SHA3 hash of the data passed in.
+// https://github.com/ethereum/eth2.0-specs/blob/master/specs/core/0_beacon-chain.md#appendix
+func Hash(data []byte) [32]byte {
+	var hash [32]byte
+
+	h := sha3.NewLegacyKeccak256()
+
+	// #nosec G104
```

Comment:
Why? Please add a comment why you ignoring this error. 

There needs to be a really good reason or you need to handle this error.

Reply:
I see from the hash interface that this never returns an error. Please add a comment as I mentioned and link to the go doc for the hash.Hash interface. This will make it easier on the security reviewer.

Reply:
Alright , I added a comment to it explaining the reason 
for not handling the error